src/Symfony/Component/Form/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -58,7 +64,7 @@ public function transform($value)
             if (!is_numeric($value)) {
                 throw new TransformationFailedException('Expected a numeric.');
             }
-            $value = (string) ($value / $this->divisor);
+            $value /= $this->divisor;
```

Comment:
I have tried forcing a rounding issue with and without casting to string and also by reverse transforming either value and I can't find any differences in the behavior. I can't remember why we put the string cast there, but I assume it was just for safety. This change looks good to me.

Reply:
@xabbuh checked the referenced issue and this fix, looks good to me, yes, doesn't seem casting to string was added on specific purpose here, so i think it should be fine.